libs/wingsdk/src/util/util.ts

```diff
@@ -135,32 +141,37 @@ export class Util {
    * @param opts `ShellOptions`, such as the working directory and environment variables.
    * @returns The standard output of the shell command.
    * @throws An error if the shell command execution fails or returns a non-zero exit code.
+   * @inflightImpl __shell__inflight
```

Comment:
nit: this reminded me of python so it looked kinda weird 🌚 Doesn't really matter but maybe a more direct convention?

```suggestion
   * @inflightImpl shell$$inflight
```

I assume a leading underscore was needed because it's marked as `@internal`, but I would also suggest just making it `private` instead. Thanks to the magic of JS, it's still accessible so it should still work.